packages/blogs-client/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gatsby-starter-hello-world",
```

Comment:
Please change the package name to @skillfuze/blogs-client so we can control it with lerna.